### DISCUSSION TITLE
Does the postcss style declaration work with tailwind 4.0

### DISCUSSION THREAD

**Discussion by aetherinsights** [tailwindlabs/tailwindcss#13485] [Help]
`<style lang="postcss">
   h1 {
        @apply text-2xl;
}
</style>
`
The above is in svelte component file. It gives me an error - cannot apply unknown utility class text-2xl. Do the postcss within the components compiled and the path is recognized? 

FYI - The @apply definition within the app.pcss or app.css works file

**Comment by alex-krasikau** [comment]
Hey @aetherinsights!

This is a known issue with an open PR for the update. Once the PR is merged and the new alpha version is released, you can expect this to work. Follow this [PR (Collect user-defined CSS that can be used in @apply)](https://github.com/tailwindlabs/tailwindcss/pull/13349) for updates.

This issue may be helpful: https://github.com/tailwindlabs/tailwindcss/issues/13305
